app/src/main/java/com/example/flightticket/LoginActivity.java

```diff
@@ -63,6 +67,8 @@ public void onClick(View view) {
                 passwordInput = loginPassword.getText().toString();
                 User loginUser = userDAO.getUserByUsername(emailInput);
                 if (validateUser(emailInput,usernames)&&validatePassword(passwordInput,loginUser.getPassword())){
+                    userPrefEditor.putString("username", emailInput);
+                    userPrefEditor.commit();
```

Comment:
It seems we only store username in SharedPreferences. How should we add flights to user? Is it possible to pass User object with SharedPreferences?

Reply:
As mentioned above, you can use the getUserByUsername() method provided by the UserDAO which will allow you get a User object from the username stored in SharedPreferences.

So to add a flight, you would get the User object from the UserDAO using the stored username, add the flight to that user object's list, and then Update() the UserDAO.